earth2studio/lexicon/goes.py

```diff
@@ -0,0 +1,74 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections.abc import Callable
+from typing import Any
+
+from earth2studio.lexicon.base import LexiconType
+
+
+class GOESLexicon(metaclass=LexiconType):
+    """GOES ABI lexicon for mapping standardized variable names to GOES ABI NetCDF variable names.
+
+    This lexicon maps our standardized spectral band names (e.g., 'vis047') to the actual
+    variable names used in GOES ABI NetCDF files on AWS. It also includes any necessary
+    data transformations or modifiers for each band.
+    """
+
+    # Mapping of standardized names to GOES ABI variable names and modifiers
+    # Format: "standardized_name": ("goes_variable_name", modifier_function)
+    # The modifier function can be used to transform the data if needed
+    GOES_VOCAB: dict[str, tuple[str, Callable[[Any], Any]]] = {
```

Comment:
Make this just `VOCAB` so its consistent with the others incase someone wants to access this directly